cmake/bintools/gnu/target_bintools.cmake

```diff
@@ -124,3 +124,10 @@ set_property(TARGET bintools PROPERTY readelf_flag_outfile ">;" )
 #set_property(TARGET bintools PROPERTY readelf_flag_headers -E)
 #set_property(TARGET bintools PROPERTY readelf_flag_infile "")
 #set_property(TARGET bintools PROPERTY readelf_flag_outfile "-O file=" )
+
+
+set_property(TARGET bintools PROPERTY nm_command ${CMAKE_NM})
```

Comment:
could you come up with a more generic name here ?
Like is done for `disassembly`, `strip`, `elfconvert`

`nm` is very GNU centric, and other toolchains may provide other commands and arguments for achieving similar functionality.
So would be nice with more generic name on what you want to achieve.